app/src/main/java/com/example/recipeopedia/models/Recipe.java

```diff
@@ -76,4 +68,57 @@ public String getInstructions() {
     public String getImage() {
         return image;
     }
+
+    public String formatString(String text){
```

Comment:
what's the gist of this method, is there nested json data that you're re-formatting?